package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "nodejs-mentoring",
+  "version": "1.0.0",
+  "description": "This repository has been created as part NodeJS mentoring program from EPAM about homeworks.",
+  "main": "app.js",
+  "scripts": {
+    "copy": "cpx ./src/config/* build/config",
+    "build": "babel ./src -d ./build",
```

Comment:
Please, setup your babel and nodemon to work together like `nodemon --exec babel-node app.js`